notebooks/week2/05_log_and_register_fe_model.py

```
@@ -0,0 +1,280 @@
+# Databricks notebook source
+# MAGIC %pip install ../power_consumption-0.0.1-py3-none-any.whl
+
+# COMMAND ----------
+
+dbutils.library.restartPython()
```

Ruff

Undefined name `dbutils`

(F821)